Assets/LeapMotion/Core/Scripts/VR/LeapVRTemporalWarping.cs

```diff
@@ -376,10 +410,10 @@ protected struct TransformData {
       Vector3 pastCenterPos = _trackingAnchor.TransformPoint(past.localPosition);
       Quaternion pastCenterRot = _trackingAnchor.rotation * past.localRotation;
 
-      transform.position = Vector3.Lerp(currCenterPos, pastCenterPos, tweenPositionalWarping);
-      transform.rotation = Quaternion.Slerp(currCenterRot, pastCenterRot, tweenRotationalWarping);
+      transform.position = Vector3.Lerp(currCenterPos, pastCenterPos, _tweenPositionalWarping);
+      transform.rotation = Quaternion.Slerp(currCenterRot, pastCenterRot, _tweenRotationalWarping);
 
-      transform.position += transform.forward * deviceInfo.focalPlaneOffset;
+      transform.position += transform.forward * _deviceInfo.forwardOffset;
```

Comment:
Is using the forward offset needed still?  What about deviceOffsetZAxis?  There seem to be two concepts here, do we actually need them both?

Reply:
We should just use deviceOffsetZAxis for everything in the LeapVRTemporalWarping script post-this-PR, I just missed it because there's a lot of code, will fix